StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/OrderingRules/UsingCodeFixProvider.cs

```diff
@@ -531,6 +560,82 @@ private List<UsingDirectiveSyntax> GenerateUsings(List<UsingDirectiveSyntax> usi
                 {
                     var currentUsing = usingsList[i];
 
+                    if (qualifyNames)
+                    {
+                        currentUsing = currentUsing.ReplaceNodes(
+                            new[] { currentUsing.Name },
+                            (originalNode, rewrittenNode) =>
```

Comment:
:thought_balloon: I would prefer this complex lambda expression inside in a method to improve readability.